crates/iota-grpc-api/proto/checkpoint.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Mysten Labs, Inc.
+// Modifications Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package iota.grpc;
+
+service CheckpointService {
+  rpc StreamCheckpoints (StreamRequest) returns (stream Checkpoint);
+  rpc GetEpochFirstCheckpointSequenceNumber (EpochRequest) returns (CheckpointSequenceNumberResponse);
+}
+
+message StreamRequest {
```

Comment:
Rename to `CheckpointStreamRequest` to avoid future clashes

Reply:
Renamed them in [65aad88](https://github.com/iotaledger/iota/pull/7511/commits/65aad88337745710d97a20f5d2c08def59a02f02).